### ISSUE TITLE
Documentation about RGB conversion: is it RGB? D65/2?

### ISSUE THREAD

**Issue by gabrielenosso** [colorjs/color-space#47]
Hi guys, I would like to know more about the rgb conversion... what does it use right now?
Does it consider any profile?

I need a conversion from sRGB (RGB D65/2) to LAB D50/2.

I also checked that when using the transformation from RGB to XYZ, it's possible to specify the parameters for the conversion... It's the RGB profile?

**Comment by dy** [comment]
Yes — `rgb` is **sRGB** (IEC 61966-2-1), white point **D65/2°**. v3 documents this explicitly and adds `lab`/`lab-d65` (D50 ICC-PCS vs display-native D65) plus arbitrary illuminants via `xyz.whitepoint` (A/C/F-series). For your sRGB→Lab **D50** need: `space.rgb.lab(...)` — in v3 `lab` is D50 by default. Will close when v3 (#67) merges.
